com.qq.QQ.yaml

```diff
@@ -8,19 +8,16 @@ command: qq
 separate-locales: false
 
 finish-args:
-  - '--share=ipc'
-  - '--share=network'
-  - '--socket=fallback-x11'
-  - '--socket=pulseaudio'
-  - '--device=all'
-  - '--filesystem=xdg-documents:ro'
-  - '--filesystem=xdg-pictures:ro'
-  - '--filesystem=xdg-videos:ro'
-  - '--filesystem=xdg-download'
-  - '--talk-name=org.freedesktop.Notifications'
-  - '--talk-name=org.kde.StatusNotifierWatcher'
-  - '--system-talk-name=org.freedesktop.login1'
-  - '--filesystem=xdg-run/pipewire-0'
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
```

Comment:
without `socket=wayland` this will never work on a wayland session.

Reply:
`fallback-x11` will add x11 if no `wayland`

Reply:
https://github.com/flathub/com.qq.QQ/pull/1

Reply:
then it should be `x11` and NOTHING ELSE.

Reply:
I think it's ok to only use `fallback-x11` or `fallback-x11 + wayland`.  
If someone add `socket=wayland` through flatseal, this app will use wayland.

Reply:
What's the reason not to add `socket=wayland` then?

Reply:
electron wayland has problem in input method, so it's should not be default.

Reply:
it should NOT be an option at all. We have been over this a few time. If wayland doesn't work out of the box, it is not an option.

It's a problem that upstream should tackle.

Reply:
So, any problem using `fallback-x11`?

Reply:
`socket=x11` is the proper way, it can only be better than `socket=fallback-x11`:
 > The fallback-x11 option makes the X11 socket available only if there is no Wayland socket.

I suppose a useless check will be avoided.

Reply:
1.This does not break any thing.  
2.Many people want to use this app with only wayland.  
3.If it's not right, flatpak should report an error when only use `fallback-x11`.  
4.We are already using 
```
export TMPDIR="$XDG_RUNTIME_DIR/app/$FLATPAK_ID"
exec zypak-wrapper /app/extra/QQ/qq $wayland_opt "$@"
```
And some apps like `com.visualstudio.code` are using more complex starting script.  
I think a wayland checking is not a problem.

Reply:
> 2.Many people want to use this app with only wayland

I doubt they want an app with a broken input

Reply:
So, it's an option, just like electron has a option to run on wayalnd.